apps/recovery-portal/src/main/webapp/self-registration-without-verification.jsp

```diff
@@ -243,10 +240,20 @@
 
                             <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 form-group">
                                 <br/>
-                                <button id="registrationSubmit"
-                                        class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large"
-                                        type="submit"><%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "Next")%>
-                                </button>
+                                <div style="display: inline-block">
+                                    <button id="registrationSubmit"
+                                            class="wr-btn grey-bg col-xs-12 col-md-12 col-lg-12 uppercase font-extra-large g-recaptcha"
```

Comment:
Are these classes still relevant?

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3298